automation_tools/__init__.py

```diff
@@ -964,6 +964,19 @@ def product_install(distribution, create_vm=False, certificate_url=None,
             host=host
         )
 
+    execute(fix_qdrouterd_listen_to_ipv6, host=host)
+
+
+def fix_qdrouterd_listen_to_ipv6():
+    """Configure qdrouterd to listen to IPv6 instead of IPv4.
+
+    Workaround for BZ #1219902.
+
+    """
+    run('sed -i -e "s|addr: 0.0.0.0|addr: ::|" '
```

Comment:
There are two instances of `addr: 0.0.0.0` in this config - one corresponds to port `5647` and the other corresonding to `5646`.  When testing, I fixed the port `5647` only.  Having said that, I am not sure if fixing both may create problem or not.  May be we can merge this and address any issues, if they occur in future.